src/utilities/methods.js

```diff
@@ -49,3 +49,16 @@ export const isDef = (val) => typeof val !== 'undefined';
 
 /** Return a generic config value if the value is undefined */
 export const renderUndefined = (value) => !isDef(value) ? Config.emptyValue : value;
+
+/** Converts a job status code to a translated string equivalent */
+export const getStatusCode = (code, t) => {
+  switch (code) {
+    case 1: return t('maintenance.jobStatus.enqueued');
```

Comment:
In some place you use t('some string variable in translate file') in some case just [this](https://github.com/Azure/pcs-remote-monitoring-webui/pull/906/files#diff-ffdbd293f00d3299797b078c0f67cf56R12) what's the difference

Reply:
In the column definitions we don't have access to the `t` method yet (since it comes through props). So what we do is define the columns with the string path in the translation file and then translate it when we need to using the `translateColumnDefs` helper method in the utilities.